src/lib/progress-circle/progress-circle.html

```diff
@@ -1,9 +1,13 @@
 <!--
   preserveAspectRatio of xMidYMid meet as the center of the viewport is the circle's
-  center.  The center of the circle with remain at the center of the md-progress-circle
+  center. The center of the circle with remain at the center of the md-progress-circle
   element containing the SVG.
 -->
 <svg viewBox="0 0 100 100"
      preserveAspectRatio="xMidYMid meet">
   <path></path>
+  <text
+    *ngIf="showText && mode === 'determinate'"
+    text-anchor="middle"
+    x="50" y="55">{{ value }}%</text>
```

Comment:
Doesn't look like it gets read out at all. Neither this one nor the regular indeterminate one.